tests/test_importance.py

```diff
@@ -15,6 +15,8 @@
 def test_importance(compute_backend_df):
     importance_vals = dd.importance(compute_backend_df, "d", return_values=True)
     assert len(importance_vals) == compute_backend_df.shape[1] - 1 - 1
+    fig = dd.importance(compute_backend_df, "d", top_features=1)
```

Comment:
When testing a different operation of a function, it should be written as a new test case (function). Only keep together if there are multiple properties/attributes that need to be checked as the result of one input.